internal/worker/leadership/tracker.go

```diff
@@ -122,6 +122,55 @@ func (t *Tracker) WaitMinion() leadership.Ticket {
 	return t.submit(t.waitMinionTickets)
 }
 
+// WithStableLeadership is part of the leadership.Tracker interface.
```

Comment:
I _hate_ these comments. 

I'd rather have the comment that's on the interface put here and a vague one on the interface than this way around.